jest.config.mjs

```diff
@@ -57,6 +57,9 @@ export default {
     '/packages/jest-haste-map/src/__tests__/haste_impl.js',
     '/packages/jest-haste-map/src/__tests__/dependencyExtractor.js',
     '/packages/jest-haste-map/src/__tests__/test_dotfiles_root/',
+    '/packages/jest-mock/src/__tests__/class-mocks-types.ts',
+    '/packages/jest-mock/src/__tests__/TestClass.ts',
+    '/packages/jest-mock/src/__tests__/SuperTestClass.ts',
```

Comment:
Minor detail. Could you move these files to `/packages/jest-mock/src/__tests__/__fixtures__/`? Recently I added `/__fixtures__/` (see above) to this list to make it less noisy.

Reply:
Sure, though I cannot see the `/packages/jest-mock/src/__tests__/__fixtures__/` folder.  The `packages/jest-mock/__typetests__/__fixtures__/` folder does not exist either though I would have expected this to be a candidate for this reorg.  Have I missed a commit/made a bad merge?

Reply:
There is no folder at the moment. Just create it (;

Reply:
I was trying to say that if you create `src/__tests__/__fixtures__/` folder it will get ignored, because `__fixtures__` was recently added to the ignore list. All would work without tweaking Jest config. That was the idea.

Reply:
Apologies, I interpreted your initial comment as being that you had already done so, somewhere.